### ISSUE TITLE
Edge Functions Don'

### ISSUE THREAD

**Issue by drewbietron** [supabase/supabase-js#719]
# Bug report

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to '…'
2. Click on '…'
3. Scroll down to '…'

## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of supabase-js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]

**Comment by dennismuench** [comment]
Yes, looks like the JS client library uses the configured SUPABASE_URL to make the request while functions are only reachable using the default domain. I have not found any Supabase CLI flags to deploy with a custom domain nor any settings in the dashboard to change the domain of a deployed function.
